bundles/CoreBundle/translations/admin.en.yaml

```diff
@@ -1,3 +1,247 @@
+    you want to continue?
+filter_by_relation: Filter by relation
+filter_by_relation_field: Filter by relation field
+clear_relation_filter: Clear active relation filter
+apply_filter: Apply filter
+config_not_writeable: >-
+    Not writable, this can have several reasons. For details please have a look
+    into the docs.
+sites: Sites
+file_exists: File already exists
+asset_upload_want_to_overwrite: Do you want to overwrite file %s?
+asset_upload_overwrite: Overwrite
+asset_upload_keep_both: Keep both files
+asset_upload_apply_to_all: Apply to all
+asset_upload_skip: Skip
+could_not_open_zip_file: The uploaded zip file could not be opened.
+objects_sort_method: Change data objects sort method
+link_recursion_error: 'The link is redirecting to itself, causing an infinite loop'
+no_further_classes_allowed: No further classes allowed
+address_not_found: The entered address was not found
+possible_causes: Possible causes
+postal_code_format_error: >-
+    Postal code format, e.g. use "5020 Salzburg, Söllheimer Straße 16" instead of
+    "A-5020 Salzburg, Söllheimer Straße 16"
+delete_data_from_it: This will also delete all data from it.
+display_mode: Display Mode
+display_mode_grid: Grid view
+display_mode_combo: Tag field
+display_mode_display: Display path reference
+display_mode_inline_search: Inline Search
+batch_confirmation: You are about to update %s dataObjects. Do you want to continue ?
+gdpr_dataSource_dataObjects: Data Objects
+parent_id: Parent ID
+modificationDate: Modification Date
+creationDate: Creation Date
+classificationstore_tag_col_group: Group
+none: None
+redirects_type_path: Path
+email_subject: Subject
+gdpr_dataSource_sentMail: Sent Emails
+email_log_subject: Subject
+gdpr_data_extractor_label_email: Email Address
+workflow_notes: Notes
+2fa_code: Code
+password: Password
+show_metainfo: Info
+email_log_from: From
+email_from: From
+log_search_from: From
+email_log_to: To
+email_to: To
+log_search_to: To
+structuredtable_type_text: Text
+objectsMetadata_type_text: Text
+gdpr_data_extractor_label_id: ID
+navigation_target: Target
+log_type: Type
+gdpr_dataSource_export: Export
+classificationstore_configuration: Configuration
+gdpr_dataSource_assets: Assets
+structuredtable_type_bool: Checkbox
+objectsMetadata_type_select: Select
+objectsMetadata_type_bool: Bool
+gdpr_data_extractor_label_firstname: Firstname
+gdpr_data_extractor_label_lastname: Lastname
+log_source: Source
+classificationstore_error_addgroup_title: Error
+classificationstore_error_addkey_title: Error
+element_lock_message: 'The desired element is currently opened by another person:'
+log_message: Message
+operator_substring_settings: Settings
+delete_gridconfig_dblcheck: Do you really want to delete this item?
+email_log_cc: Cc
+email_log_bcc: Bcc
+cols: Columns
+log_search_message: Message
+header: Header
+no_header: No Header
+batch_export_confirmation: You are about to export %s dataObjects. Do you want to continue ?
```

Comment:
These translations have been moved to admin bundle (https://github.com/pimcore/pimcore/commit/736d45a95ac3a1ab2b067950a46d71b8f6bf5d35)